guidance/models/_model.py

```diff
@@ -862,6 +870,8 @@ def __init__(self, engine, echo=True, **kwargs):
         self._event_parent = None
         self._last_display = 0  # used to track the last display call to enable throttling
         self._last_event_stream = 0  # used to track the last event streaming call to enable throttling
+        self._state_html_parser = ModelStateHTMLParser() # used to parse the state for cli display
```

Comment:
Same as before, consider making naming specific to console if it's only used for console mode.